TwoNote/README.md

```diff
@@ -1,8 +1,67 @@
-# Introduction 
-This repo contains the samples for Surface Duo 
+---
+page_type: sample
+name: Surface Duo - TwoNote
+description: "Highlights pen events, drag and drop, and multiple dual-screen app patterns to enhance a notes app for the Surface Duo."
+languages:
+- kotlin
+products:
+- surface-duo
+urlFragment: TwoNote
+---
+
+# TwoNote
+
+This sample integrates various dual-screen app features into a note-taking app for the Surface Duo. Users can view and edit notes via the [list-detail](https://docs.microsoft.com/dual-screen/introduction#master-detail) or [extended canvas](https://docs.microsoft.com/dual-screen/introduction#extended-canvas) dual-screen app patterns. Each note may contain text and images, which can both be imported through drag/drop. In addition, users can add drawings and annotations to their notes with pen events.
 
 ## Getting Started
-To learn how to load your app on to emulator, and to use it ,see the full documentation here: [https://docs.microsoft.com/dual-screen](https://docs.microsoft.com/dual-screen)
+
+To learn how to load apps on the Surface Duo emulator, see the [documentation](https://docs.microsoft.com/dual-screen/android), and follow [the blog](https://devblogs.microsoft.com/surface-duo).
+
+## Features
+
+In addition to including various dual-screen enhancements, this app supports a range of operations related to notes, note categories, and note contents.
+
+### Notes
+
+Notes can be created by clicking the floating action button in the list view. To rename a note, click on its current title and make the desired changes. To delete a note, users can choose `Delete` from the overflow menu in the detail view of the note. To delete multiple notes at once, long-click on notes or choose `Select all` from the overflow menu in the list view. To share an individual note as an image, scroll to the desired section of the note when in the detail view and choose the `Share` option.
```

Comment:
we should probably replace `scroll to the desired section of the note...` with `choose the 'Share' option when in detail view`